tests/alloc_with.rs

```diff
@@ -0,0 +1,65 @@
+// All of these alloc_with tests will fail with "fatal runtime error: stack overflow" unless LLVM
+// manages to optimize the stack writes away.
+
+#[test]
+#[cfg(not(debug_assertions))]
```

Comment:
Instead of writing this `cfg` on every item in this file, you can add this to the top of the file:

```rust
#![cfg(not(debug_assertions))]
```